backend/src/zango/apps/shared/platformauth/templates/app_panel/app_panel_login.html

```diff
@@ -103,10 +103,9 @@
     </div>
     <div class="footer"
       style="position: absolute; bottom: 32px; display: flex; justify-content: space-around; right:0; left:0; margin:0;">
-      <div class="footer-Copyright">Copyright © <span id="getYear"></span> HEALTHLANE TECHNOLOGIES PVT. LTD</div>
       <div class="footer-content">
-        <a href="https://www.zelthy.com/privacy/" class="footer-term" target="_blank" rel="noopener">Terms</a>
-        <a href="https://www.zelthy.com/terms/" class="footer-privacy" target="_blank" rel="noopener">PRIVACY</a>
+        <a href="https://github.com/Healthlane-Technologies/Zango/blob/main/LICENSE" class="footer-license-" target="_blank" rel="noopener">License</a>
+        <a href="https://www.zango.dev/docs/" class="footer-docs" target="_blank" rel="noopener">Docs</a>
```

Comment:
@gouravanirudh05 Since the css classes were renamed we also need to rename them in the css file which is being used here, please change it there also and make sure to review it in your local once

Reply:
@gouravanirudh05 did you get a chance to look at the changes requested? Please let us know if you have other priorities and cannot work on this at this moment